framework/src/modules/chain/submodules/loader.js

```diff
@@ -813,21 +813,28 @@ __private.loadBlocksFromNetwork = function(cb) {
 						modules.blocks.process.loadBlocksFromNetwork(
 							(loadBlocksFromNetworkErr, lastValidBlock) => {
 								if (loadBlocksFromNetworkErr) {
+									library.logger.error(
+										loadBlocksFromNetworkErr instanceof Error
```

Comment:
`convertErrorsToString` in the helper can be used for this purpose

Reply:
We want to pass an error as a first argument to the logger for Bunyan whereas `convertErrorsToString`  always returns a string. Otherwise, it will simply concatenate stringified error with the message in the second argument.